generator-scrivito/generators/widget/index.js

```diff
@@ -0,0 +1,43 @@
+const Generator = require("yeoman-generator");
+const lodash = require("lodash");
+const prettier = require("gulp-prettier");
+
+module.exports = class extends Generator {
+  start() {
+    this.prompt({
+      type: "input",
+      name: "widgetClassName",
+      message: "Enter a name of the Widget (e.g. GiphyWidget):",
+    }).then(answers => {
+      const widgetClassName = answers.widgetClassName;
+      const folder = `src/Widgets/${widgetClassName}`;
+      const humanFriendlyName = lodash.startCase(
+        widgetClassName.replace(/Widget$/, "")
+      );
+      this._generateFile(
+        "XWidgetComponent.js.ejs",
+        `${folder}/${widgetClassName}Component.js`,
+        { widgetClassName }
+      );
+      this._generateFile(
+        "XWidgetEditingConfig.js.ejs",
+        `${folder}/${widgetClassName}EditingConfig.js`,
+        { widgetClassName, humanFriendlyName }
+      );
+      this._generateFile(
+        "XWidgetClass.js.ejs",
+        `${folder}/${widgetClassName}Class.js`,
+        { widgetClassName }
+      );
+    });
+  }
+
+  _generateFile(templatePath, destinationPath, context) {
+    this.fs.copyTpl(
+      this.templatePath(templatePath),
+      this.destinationPath(destinationPath),
+      context
+    );
+    this.registerTransformStream(prettier());
```

Comment:
This needs to be set _before_ calling `copyTpl`. I suggest to move `this.registerTransformStream(prettier());` to the `start` method instead (e.g. as the very first command executed):

```js
  start() {
    this.registerTransformStream(gulpPrettier());

    this.prompt({
      type: "input",
      name: "objClassName",
      message: "Enter the name of the Obj (e.g. FaqPage):",
    }).then(answers => {
    // ...
```